about.html

```diff
@@ -68,6 +68,14 @@ <h3 style="display:inline">Changelog</h3>
         Show
       </button>
       <ul id="changelog" style="display:none">
+        <li class="versionnumber">0.532.5</li>
+        <li>
+          Fixed this changelog to accurately reflect how many locations were actually removed (four, not two).
+        </li>
+        <li class="versionnumber">0.532</li>
+        <li>
+          Fixed four incorrect starting locations in the Permanant Elusi- er.."Special Assignments" (If you're curious, it was the "Shaman's Hut" and "Village Bus Stop" starting points in Columbia, and the "Chawls" and "Photoshoot" starting points in Mumbai)
```

Comment:
Just call them special assignments, no need to be a smartass here.

Reply:
doneso.